src/components/tasks/card/index.tsx

```diff
@@ -34,7 +34,16 @@ const Card: FC<Props> = ({
   const iconHeight = '25px';
   const iconWidth = '25px';
 
-  const cardClassNames = [classNames.card];
+function formattedDates(localEndsOn: Date){
+  const year = localEndsOn.getFullYear()
+  const month = localEndsOn.getMonth() + 1;
+  const day = localEndsOn.getDate();
+  const finalDate = [year, month, day].join('-');
+  return finalDate;
+}
+
+  const dates = !!localEndsOn ? formattedDates(localEndsOn) : '';
+  const [dateTimes, setDateTimes] = useState(dates);
```

Comment:
Few Suggestions for this Function : 

1. move this function in a util.
2. function name can be improved to something like getDateInStringFormat
3. parameter to the function can be named date instead of localEndsOn
4. finalDate can be named better something like strDate.